database/psql.js

```diff
@@ -0,0 +1,247 @@
+const inputs = require('../server/inputs.js');
+const {getTrialEndDate} = require('../helpers/getTrialEndDate.js');
+const Sequelize = require('sequelize');
+const sequelize = new Sequelize({
+  database: 'test',
+  dialect: 'postgres',
+  username: 'abscura',
+  password: null,
+  operatorsAliases: Sequelize.Op,
+});
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('\x1b[32m----- SUCCESSFUL DATABASE CONNECTION -----\x1b[0m');
+  })
+  .catch(err => {
+    console.error('\x1b[31m----- FAILED DATABASE CONNECTION -----\x1b[0m');
+    console.error(err);
+  });
+
+
+/* -------------------- TABLE DEFINITION -------------------- */
+
+
+const Users = sequelize.define('users',
+  {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      unique: true,
+      autoIncrement: true
+    },
+    first_name: { type: Sequelize.STRING },
+    last_name: { type: Sequelize.STRING },
+    email: { type: Sequelize.STRING },
+    contact_number: { type: Sequelize.STRING }
+  },
+  { timestamps: false }
+);
+
+const Addresses = sequelize.define('addresses',
+  {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      unique: true,
+      autoIncrement: true
+    },
+    user_id: { type: Sequelize.STRING },
```

Comment:
You might need to import user_id as an integer since you define it above as an integer, assuming these two columns are going to be used to join the users and addresses tables together.

Reply:
Good eye! `user_id` should definitely be an integer. Fixed in upcoming PR!